Automattic/jetpack
# Social: Use Blog ID in links to WPCOM instead of site slug

## Proposed changes:
Adjust checkout links in Social plugin to use blog ID instead of slug for multi-URL support.

### Other information:


## Jetpack product discussion
pf5801-93-p2

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
To test the blog, you'll need two domains pointing to the same site. One way to do that is to setup two proxying tunnels to the same local environment.

**Connect Jetpack with a free plan on URL A, and load it via URL B.**

If your site falls into IDC due to concurrency issues with IDC secret validation (fix pending), remove `jetpack_sync_error_idc` option.

1. Using URL B (not the cache site's main URL), visit "Jetpack -> Social" and purchase a paid plan.
2. After purchase, you should be redirected to "Jetpack -> Social" on URL A (cache site's main URL).
3. Go back to URL B's "Jetpack -> Social" and click on the Jetpack Could links to confirm they lead you to the correct page:
    - "Connect Accounts"
    - Manage social media accounts"
